pki/pki.go

```diff
@@ -950,7 +960,11 @@ func (p *PKI) Save(opt ...ConfigOption) error {
 			return errs.FileError(err, p.defaults)
 		}
 		if p.profileDefaults != "" {
-			if err = fileutil.WriteFile(p.profileDefaults, []byte("{}"), 0644); err != nil {
+			if _, err := os.Stat(p.profileDefaults); os.IsNotExist(err) {
+				if err = fileutil.WriteFile(p.profileDefaults, []byte("{}"), 0644); err != nil {
```

Comment:
It's weird to see 0644 where we always create everything as 0600/0700.
```suggestion
				if err = fileutil.WriteFile(p.profileDefaults, []byte("{}"), 0600); err != nil {
```

Reply:
The `$(step path)/config/defaults.json` are written with `0644  right below this.  Should those also be `0600`? Why are those different?